gogs_client/interface.py

```diff
@@ -42,6 +42,68 @@ def authenticated_user(self, auth):
         response = self._get("/user", auth=auth)
         return GogsUser.from_json(self._check_ok(response).json())
 
+    def get_tokens(self, auth, username=None):
+        """
+        Returns tokens defined for specified user.
+        If no user specified uses user authenticated by the given authentication.
+        Right now, authentication must be UsernamePassword (not Token).
```

Comment:
Why must `auth` be `UsernamePassword`? `self.authenticated_user(..)` and `self._get(..)` should be able to handle any authentication type.